Doc/library/pyexpat.rst

```diff
@@ -321,6 +321,11 @@ callback).
    Current byte index in the parser input.
 
 
+.. attribute:: xmlparser.CurrentByteCount
```

Comment:
I would document this before `xmlparser.CurrentByteIndex` to keep the attributes alphabetically ordered.

Reply:
Good point. I didn't catch that we weren't following the order of the expat documentation.